ee/modules/500-operator-trivy/templates/deployment.yaml

```diff
@@ -79,6 +79,8 @@ spec:
             value: "false"
           - name: OPERATOR_SCAN_JOB_TIMEOUT
             value: "5m"
+          - name: OPERATOR_SCAN_JOB_REGISTRY_CA
+            value: | {{ $.Values.global.modulesImages.registry.CA | nindent 14 }}
```

Comment:
idk why, but I don't have this property set in my cluster even if I switch to self-signed issuer :thinking: 
![image](https://github.com/deckhouse/deckhouse/assets/41360396/5852cb52-78fe-4947-ae39-314ce2b13a3f)
any ideas how it came about?
Moreover, I'm afraid of a hypothetical situation when a client might use several self-signed certificates for different parts of their infrastructure, so that the CA certificate of deckhouse cluster doesn't equal to their registry CA (an external registry like a nexus instance).
Could it be better to provide a way of specifying any custom CA via the module's properties, falling back to default value from `.global.modulesImages.registry.CA` if it isn't set?